enclave_manager/Dockerfile

```diff
@@ -160,8 +160,6 @@ ARG MAKECLEAN
 ARG TCF_DEBUG_BUILD
 
 # Environment setup
-ENV http_proxy=$http_proxy
-ENV https_proxy=$https_proxy
```

Comment:
Have we tested SGX h/w with this change. Please verify if connection to IAS server works after removing this.